app/views/pics/index.html.erb

```diff
@@ -0,0 +1,39 @@
+<h2>The feed</h2>
+
+<%= link_to "Post a pic", new_pic_path %>
+
+<div id="the-feed">
+  <% @pics.each do |pic| %>
+    <div class="pic">
+      <div class="pic-poster">
+        <% if pic.user %>
+          This pic posted by <%= pic.user.email %>
+        <% else %>
+          This pic posted by unknown?!
+        <% end%>
+      </div>
+      <img src="<%= url_for(pic.image) %>" >
+      <div class="comments">
+        <% pic.comments.each do |comment| %>
+          <div class="a-comment">
+            <div class="comment-author">
+              <% if comment.user %>
+                This judgement was passed by <%= comment.user.email %>:
```

Comment:
Adding 'delegate :username to: :user' to the Comment model will mean that you don't have to reach through User class to get to username. One of the other SOLID principles apparently that isn't SRP.